src/transformState.js

```diff
@@ -4,8 +4,28 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
   // write code here
-}
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
 
+      case 'removeProperties':
+        for (const names in action.keysToRemove) {
+          delete state[action.keysToRemove[names]];
```

Comment:
If you use `for of `loop it will simply look like 

```suggestion
        for (const names of action.keysToRemove) {
          delete state[names];
```